cmd/create/cluster/cmd.go

```diff
@@ -912,11 +912,15 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
-	// FIXME: Remove before GA
 	if isHostedCP && r.Reporter.IsTerminal() {
-		//nolint
-		r.Reporter.Infof("NOTE: Hosted control planes are currently in Technology Preview (https://access.redhat.com/support/offerings/techpreview)." +
-			" Any Technology Preview clusters will need to be destroyed and recreated prior to general availability.")
+		techPreviewMsg, err := r.OCMClient.GetTechnologyPreviewMessage(ocm.HcpProduct, time.Now())
+		if err != nil {
+			r.Reporter.Errorf("%s", err)
+			os.Exit(1)
+		}
+		if techPreviewMsg != "" {
+			r.Reporter.Infof(techPreviewMsg)
+		}
```

Comment:
Just wondering, what if the techPreviewMessage = ""? I guess we just skip that scenario?

Reply:
Yes, if the message is empty, there is no print, then the ROSA CLI output won't have that extra line.